listing-description/en-US.md

```diff
@@ -2,39 +2,35 @@
 
 A Fabric-based Minecraft modpack focusing on performance and graphics enhancements.
 
-[![Current review](https://img.youtube.com/vi/bb8G9X5Q_4I/hqdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
+[![Video review](https://img.youtube.com/vi/bb8G9X5Q_4I/hqdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
 
 ### Performance
 
-Fabulously Optimized is usually faster than OptiFine thanks to [modern performance mods][1] such as Sodium, Lithium and more!
+Fabulously Optimized is faster than OptiFine thanks to [modern performance mods][1] such as Sodium, Lithium and others! The bundled mods have been carefully selected to be safe to your worlds and bring noticeable speed benefits on any device.
 
 ### OptiFine parity
 
-Fabulously Optimized [supports almost all OptiFine features][2], including shaders, connected textures and zoom. [You can even get your own free cape!][3]
+Fabulously Optimized [supports most OptiFine features][2], including shaders, connected textures and zoom. If your resource pack says "OptiFine required", it will likely work on here as well. [You can also get a free cape!][3]
 
-### Minor tweaks
+### Feels familiar
 
-Fabulously Optimized [includes a few minor tweaks][4] that don't break the vanilla feel, such as extended chat history, windowed fullscreen mode, fixes to most common bugs and utilities to diagnose crashes.
+Fabulously Optimized [includes features that let you play Minecraft just as you remember it][4], no matter what version you've used before. That means you can use game controllers, play local worlds with friends, see advancements in full screen and more. 
 
 ### Works everywhere
 
-Fabulously Optimized has no content mods, so it works on any world or server. The modpack [can be run on 4 launchers, is hosted on 3 sites][6] and [is translated to many languages][7].
+Fabulously Optimized has no content mods, so you can play on any world or server. While you can play the pack with any game launcher, [we encourage and support these as they give you the best experience.][6]
 
 ### Up to date
 
-Fabulously Optimized [receives fast updates][5] and is one of the first modpacks to upgrade to newer Minecraft versions.
-
-### Built on Fabric
-
-Compared to Forge, Fabric is lightweight, updates and loads fast, and still has great mod support.
+Thanks to Fabric mod loader, Fabulously Optimized [receives fast updates][5] and is one of the first modpacks to upgrade to newer Minecraft versions. [Updating the modpack is easy][17] and will preserve your configuration changes, where possible.
```

Comment:
I would emphasize more the fact that fo is available soon after the release, faster than optifine. Also with these changes the "Thanks to fabric loader" doesn't make much sense here.

I think a new title like this fits well: 
`### Built on Fabric

Fabric is a lightweight, fast, and up-to-date mod loader that has huge mod support.
You can use Fabulously Optimized for modded Minecraft gameplay (and as a base for your own modpack, thanks to his open-source license).
`
the between-parenthesis part is optional.

Reply:
```suggestion
Thanks to the Fabric mod loader, Fabulously Optimized [receives swift updates][5] and stands as one of the first modpacks to upgrade to newer Minecraft versions. [Updating the modpack is a straightforward process][17] and will preserve your configuration changes, where possible.
```

Reply:
Re-added the separate Fabric paragraph and rewrote both.